process/logsevents/pedingBalances.go

```diff
@@ -0,0 +1,35 @@
+package logsevents
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/converters"
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+)
+
+type pendingBalancesProc struct {
+	pendingBalances map[string]*data.AccountInfo
+}
+
+func newPendingBalancesProcessor() *pendingBalancesProc {
+	return &pendingBalancesProc{
+		pendingBalances: make(map[string]*data.AccountInfo),
+	}
+}
+
+func (pbp *pendingBalancesProc) addInfo(receiver string, token string, tokenNonce uint64, value string) {
+	hexEncodedNonce := converters.EncodeNonceToHex(tokenNonce)
+	key := fmt.Sprintf("%s-%s-%s-%s", pendingBalanceIdentifier, receiver, token, hexEncodedNonce)
```

Comment:
couldn't you use the token identifier computer and only append the `pending_` prefix ?
Also, I would recommend `pending_<token id>` instead of `pending-<token-id>` which also contains hypens

Reply:
In this case, I will let as it.
1. In all places I sue "-" instead of "-"
2. Method from converters.ComputeTokenIdentifier will not work for fungible ESDT ( nonce = 0 )